### ISSUE TITLE
Image appears twice when changing the alignment.

### ISSUE THREAD

**Issue by mauritsvanrees** [plone/Products.CMFPlone#3800]
Plone 6.0.5 (coredev).

* Create a Classic UI Plone Site.
* Add a Page.
* Edit the page, upload an image, align it to the right, save the page.
* Edit the page again, select the image, open the image model, align it on the left.
* The image is now visible twice.

HTML:

```
<p>
<img
 class="image-richtext image-left picture-variant-medium"
 src="../resolveuid/1e08b0c0be6a4d66bf20491a88430a48/@@images/image/teaser"
 alt=""
 data-linktype="image"
 data-scale="teaser"
 data-val="1e08b0c0be6a4d66bf20491a88430a48"
 data-picturevariant="medium" />
<img
 class="image-richtext image-right picture-variant-medium"
 src="../resolveuid/1e08b0c0be6a4d66bf20491a88430a48/@@images/image/teaser"
 alt=""
 data-linktype="image"
 data-scale="teaser"
 data-val="1e08b0c0be6a4d66bf20491a88430a48"
 data-picturevariant="medium" />
</p>
```

**Comment by petschki** [comment]
I could not reproduce this on classic demo site and local with latest staticresources.

**Comment by mauritsvanrees** [comment]
Fixed in Plone 6.0.10.
